### PR TITLE
chore: update actions

### PR DESCRIPTION
**Pull request by Michiel87**

### CODE REVIEW

File: dummy-data/src/index.ts [hunk 1]
```diff
@@ -80,7 +80,7 @@ async function installComponent(
     octokit,
     owner: 'exivity',
     repo: component,
-    ref: 'master',
+    ref: STANDARD_BRANCH,
```
Comment (Michiel87) [inline dummy-data/src/index.ts:83]: @CalliEve - All components now only have a main branch (apart from scaffold), so I renamed this, correct?

File: lib/github.ts [hunk 2]
```diff
@@ -24,32 +24,9 @@ type Options = {
...
-    ).data.map((branch) => branch.name)
-    if (!availableBranches.includes('develop')) {
-      const fallback = availableBranches.includes('main') ? 'main' : 'master'
-      warning(
-        `Branch "develop" not available in repository "${owner}/${repo}", falling back to "${fallback}".`
-      )
-      ref = fallback
-    }
-  }
+export const STANDARD_BRANCH = 'main'
```
Comment (Michiel87) [inline lib/github.ts:27]: @hongaar - I removed this bit from getShaFromRef as all components now just use a main branch. Is this ok?

Comment (Michiel87) [reply lib/github.ts:27]: see function below this comment

File: db/src/index.ts [hunk 3]
```diff
@@ -1,22 +1,21 @@
...
 async function run() {
   // Input
-  const branch =
-    getInput('branch') ||
-    (context.ref === 'refs/heads/main'
-      ? 'main'
-      : context.ref === 'refs/heads/master'
-      ? 'master'
-      : 'develop')
+  const branch = getInput('branch') || STANDARD_BRANCH
```
Comment (Michiel87) [inline db/src/index.ts:18]: @hongaar  - this is also fine now right?

File: get-artefacts/src/index.ts [hunk 4]
```diff
@@ -1,20 +1,14 @@
...
   const component = getInput('component', { required: true })
   let sha = getInput('sha')
-  const branch =
-    getInput('branch') ||
-    (context.ref === 'refs/heads/main'
-      ? 'main'
-      : context.ref === 'refs/heads/master'
-      ? 'master'
-      : 'develop')
+  const branch = getInput('branch') || STANDARD_BRANCH
```
Comment (Michiel87) [inline get-artefacts/src/index.ts:11]: @hongaar - all good here?

**Review by CalliEve — APPROVED** [review]

File: dummy-data/src/index.ts (hunk 1 above)
Comment (CalliEve) [reply dummy-data/src/index.ts:83]: yeah should be fine!

File: accept/src/index.ts [hunk 5]
```diff
@@ -151,10 +150,15 @@ async function run() {
     }
   }
 
-  if (eventName === 'pull_request') {
+  if (isEvent(eventName, 'pull_request', eventData)) {
     // We need to check if required workflow has finished
     info('Checking if workflow constraint is satisfied...')
 
+    if (eventData['action'] !== 'ready_for_review') {
+      warning(`[accept] Skipping: workflow constraint not satisfied`)
```
Comment (hongaar) [inline accept/src/index.ts:158]: not: update debug message

File: db/src/index.ts (hunk 3 above)
Comment (hongaar) [reply db/src/index.ts:18]: LGTM

File: get-artefacts/src/index.ts (hunk 4 above)
Comment (hongaar) [reply get-artefacts/src/index.ts:11]: 👍

**Review by hongaar — COMMENTED** [review]
One nit, looks good otherwise 💪

**Pull request merged**